packages/extension/src/ui/action/views/network-activity/components/network-activity-transaction.vue

```diff
@@ -141,6 +149,11 @@ const getFiatValue = computed(() => {
 });
 onMounted(() => {
   date.value = moment(props.activity.timestamp).fromNow();
+  chainId.value =
+    props.activity.network === NetworkNames.Kadena ||
+    props.activity.network === NetworkNames.KadenaTestnet
+      ? props.activity.chainId
+      : undefined;
```

Comment:
we dont prefer having customized options specific to certain networks.
you can do something like `v-if="network.subNetworks"` and display the chainid

Reply:
That was my first approach but I made it specific in case other networks have subnetwork (so as not to affect their views). I see that they do not though, so I will change it to your suggestion

Reply:
I made this change (and also removed some code that belonged in other PRs.)